crates/bevy_render/src/camera/projection.rs

```diff
@@ -134,54 +134,71 @@ impl Default for PerspectiveProjection {
     }
 }
 
-// TODO: make this a component instead of a property
-#[derive(Debug, Clone, Reflect, FromReflect, Serialize, Deserialize)]
-#[reflect(Serialize, Deserialize)]
-pub enum WindowOrigin {
-    Center,
-    BottomLeft,
-}
-
 #[derive(Debug, Clone, Reflect, FromReflect, Serialize, Deserialize)]
 #[reflect(Serialize, Deserialize)]
 pub enum ScalingMode {
-    /// Manually specify left/right/top/bottom values.
+    /// Manually specify the projection's size with `width` and `height`.
     /// Ignore window resizing; the image will stretch.
-    None,
-    /// Match the window size. 1 world unit = 1 pixel.
-    WindowSize,
-    /// Use minimal possible viewport size while keeping the aspect ratio.
+    None { width: f32, height: f32 },
+    /// Match the window size.
+    /// The argument is the number of pixels that equals one world unit.
+    WindowSize(f32),
+    /// Use the minimal possible projection size while keeping the aspect ratio.
```

Comment:
Can you explain what is meant by "projection size"? Is this the same as the frustum's cross-section?

Reply:
Yes. I think using "projection" as the plane that the frustum expands from makes it a bit simpler (rather than having to specify *cross sectional* area every time). I find it fairly intuitive but if you have any suggestions that's more than welcome.

Reply:
@xgbwei We can use `projection`, but then we should define it first. The issue is can mean multiple things in different contexts. (Also, what is intuitive to one reader might not be intuitive to another; that depends on the experiences the reader might have had.)

I think we should have a sentence that defines `projection`, and connects it to the idea of `cross-sectional area`, and then we can use projection. I will try and provide a suggestion.

Reply:
@xgbwei We can use `projection`, but then we should define it first. The issue is can mean multiple things in different contexts. (Also, what is intuitive to one reader might not be intuitive to another; that depends on the experiences the reader might have had.)

I think we should have a sentence that defines `projection`, and connects it to the idea of `cross-sectional area`, and then we can use `projection`. I will provide a suggestion.

Reply:
I'm wondering if we need to take into account the near and far clipping planes as part of the "shape" of the projection. Clipping planes are mostly there for technical reasons, they aren't inherently part of orthographic projection.

Reply:
That being said, they should be taken into account when describing the view frustum, since frustum culling culls objects outside of the near and far clipping planes.